Annotation/Arg.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sensio\Bundle\FrameworkExtraBundle\Annotation;
+
+/**
+ * @Annotation
+ * @Target({"METHOD"})
+ * @Attributes({
+ *     @Attribute("value", type="string", required=true),
+ * })
+ */
+class Arg
+{
+    private $arg;
```

Comment:
It's still a pretty cool extension point and used by some packages. Anyway, this is probably a challenge for later on.

https://github.com/FriendsOfSymfony/FOSHttpCacheBundle/blob/master/Configuration/InvalidatePath.php
https://github.com/jaytaph/RateLimitBundle/blob/master/Annotation/RateLimit.php
https://github.com/mayeco/EasyAnnotationBundle

Google `in:github.com "extends ConfigurationAnnotation"`